api-catalog-ui/frontend/cypress/e2e/detail-page/swagger-try-out-and-code-snippets.cy.js

```diff
@@ -24,12 +24,6 @@ describe('>>> Swagger Try Out and Code Snippets Test', () => {
     });
 
     [
-        {
```

Comment:
what was the reason of removing this?

Reply:
I don't see a reason to test the mock service, is there some benefit?

Reply:
I put zosmf back